utils/src/main/java/com/nicehash/utils/cli/IoUtils.java

```diff
@@ -0,0 +1,21 @@
+package com.nicehash.utils.cli;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class IoUtils {
+
+    public static void copyBytes(InputStream is, OutputStream os, boolean flush) {
+        int c;
+        try {
+            while ((c = is.read()) != -1) {
+                os.write(c);
+                if (flush) {
```

Comment:
Do you really want to flush after every write?

Reply:
Sometimes yes, that's why there is 'flush' argument. This is used for example when copying output of websocket connection to stdout in real time. Since stdout is buffered, written bytes aren't actually flushed to native stdout until newline character or until flush is called explicitly. Not calling flush in that case results in incomplete output.